docs/content/docs/reference/telemetry.md

```diff
@@ -24,7 +24,7 @@ The questions we want to currently answer using telemetry are the following:
 
 - How many active developers are using Keystone, compared to how many are active in the community,
 - How complex are projects that use Keystone,
-- What field types and `@keystone-6` or community packages are being used by developers, and
+- What field types and `@keystone-6/` or community nominated packages are being used by developers, and
```

Comment:
I have emphasised `nominated` here, in the event other community package authors want to include their packages, we are open to adding them!